bftengine/include/bftengine/ControlStateManager.hpp

```diff
@@ -0,0 +1,31 @@
+// Concord
+//
+// Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0
+// License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+
+#pragma once
+#include <string>
+#include <unordered_map>
+#include "IStateTransfer.hpp"
+
+namespace bftEngine {
+
+class ControlStateManager {
```

Comment:
Do we want to have the ability to copy the instance of the object?
I mean something like:
```
std::shared_ptr< ControlStateManager> m;
auto m1 = *m;
```
If not, it might be better to delete copy/assign operators.